versions/1.0.0.md

```diff
@@ -8,7 +8,12 @@ This document is licensed under [The Apache License, Version 2.0](https://www.ap
 
 ## Introduction
 
-TBD
+The Overlay specification is an extension or companion to the OpenAPI specification.
+An Overlay describes a set of changes to be applied or "overlaid" onto an existing OpenAPI document.
```

Comment:
Like with OAS, we'll probably need a way to differentiate between Overlay Specification, an Overlay document, and the conceptual manifestation that may be a singular document or a cohesion of documents (Overlay Description?). I would recommend adopting the patterns we have with OpenAPI already if they make sense, but I'm open to other opinions on this.

Reply:
I'm not sure what you're suggesting in terms of rewording, can you clarify if there's a change you had in mind?

I used "Overlay" pretty intentionally, but I now think the end should be "OpenAPI description" and I'll update.

Reply:
We don't have the `$ref` problems here that requires the distinction between a "document" (single file) and "description" (one or more files linked via `$ref`, so it would suffice to always write either "Overlay Specification" or "Overlay document" and avoid stand-alone occurrences of "Overlay" or "Overlays".

Have applied that as suggestions here.

If accepted, we need another pass over the spec to adjust places not changed by this PR.